### PR TITLE
add cooper black font to project

### PR DESCRIPTION
**Pull request by aholtzman**
## Related Issue

resolves issue #32 
## Summary of Changes

1. [x] added font file
1. [x] loaded font file in theme.css

### CODE REVIEW

**Comment by thescientist13** [comment]
@aholtzman 
Do you mind re-pointing to `v1` branch?  That way it can get into the mainline for everyone, since I still have a couple more TODOs on that branch.  (no big deal if not, I can always re-point)

File: src/styles/theme.css [hunk 1]
```diff
@@ -2,6 +2,11 @@
 @tailwind components;
 @tailwind utilities;
 
+@font-face {
+  font-family: 'Cooper Black';
+  src: url('../assets/COOPBL.woff2') format('woff2');
```
Comment (thescientist13) [inline src/styles/theme.css:7]: Only tweak that comes to mind (and I don't mind adding myself) is maybe trying to add [`font-display: swap`](https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/font-display#values) as I _think_ it makes the transition or ["popping" in of fonts](https://developer.chrome.com/blog/font-display/#which-font-display-is-right-for-you) a little more subtle.

Not sure if you have any thoughts / experience.

File: src/styles/theme.css [hunk 2]
```diff
@@ -2,6 +2,12 @@
 @tailwind components;
 @tailwind utilities;
 
+@font-face {
+  font-family: 'Cooper Black';
+  src: url('../assets/coopbl.woff2') format('woff2');
```
Comment (thescientist13) [inline src/styles/theme.css:7]: just a heads up the linter **ls-lint**) is checking the _filename_ itself, so both should match. 👍

Comment (aholtzman) [reply src/styles/theme.css:7]: for some reason git was not registering the change is case on the file so it never got pushed. I will try a new name now.

**Review by thescientist13 — APPROVED** [review]
Awesome, thank you!  🙏

**Pull request merged**